pandas/core/strings.py

```diff
@@ -1316,6 +1358,27 @@ def str_encode(arr, encoding, errors="strict"):
     return _na_map(f, arr)
 
 
+def str_normalize(arr, form):
```

Comment:
Nearly all the StringMethods methods follow a pattern of defining `str_foo` as a module-level function and then wrapping it as the `foo` method.  `StringMethods.normalize` didn't follow this pattern, so I moved it out.  At the time I intended to use `_delegate_method` to get rid of a bunch of boilerplate in the class (and clarify its relation to other `PandasDelegate` subclasses), but eventually chose to hold off on that.

So this isn't strictly pertinent to this PR; I'll be happy to revert it.